### ISSUE TITLE
How to set .xmind file use XMind to open and change the file icon?

### ISSUE THREAD

**Issue by abcfy2** [mriza/XMind-Linux-Installer#15]
First I would say this is a nice script. The only one issue is that would you please set the `.xmind` file use `XMind` open as default ? And how to change the `.xmind` file icon instead of text file?

![image](https://user-images.githubusercontent.com/5327677/45401142-d2e41700-b681-11e8-97d2-3fdb2423f239.png)

As above, how do I change the icon from text file to xmind icon?

**Comment by fmorato** [comment]
Mimetype should be defined in the .desktop file to be the same as defined in the xmind.xml file.

**Comment by mriza** [comment]
yes, @fmorato is right. unfortunately I don't have time to implement it for now. Maybe if I have some spare time. Thank you
